src/components/CharacterInfo.vue

```diff
@@ -11,8 +11,31 @@ import CharacterInfoPortrait from "./CharacterInfoPortrait.vue";
  */
 export default {
   components: [CharacterInfoInput, CharacterInfoSelect, CharacterInfoPortrait],
-  props: ["bio", "generations"],
+  props: ["bio", "predatorDefinitions"],
   emits: ["bioChange"],
+  computed: {
+    /**
+     * @returns list of disciplines filtered by predator type value from predatorDefinitions
+     */
+    predatorTypeDisciplines() {
+      return this.predatorDefinitions.filter(
+        (predator) => predator.id == this.bio.predator.value
+      )[0].disciplines
+    }
+  },
+  methods: {
+    /**
+     * after change of predator type check if we have to remove predator discipline
+     */
+    chosenDisciplineCheck(predatorType) {
+      var allowedDisciplines = this.predatorDefinitions.filter(
+        (predator) => predator.id == predatorType
+      )[0].disciplines
+      if (! allowedDisciplines.includes(this.bio.predator.chosenDiscipline)) {
```

Comment:
a potom tam ostane len tato podmienka a neviem ako casto sa to pouziva, teoreticky by to islo vrazit do kodu. alebo mozno tam pouzit tu `vec = podmienka ? podmZmena : bezZmeny` alebo nieco take, ale asi to je jedno asi mi tam vadila ta computed, hned ma to proste nejak prastilo do oka, ze je to divne

Reply:
tam neni else, takze neviem ako by tam bola :